src/toxcore/group/packet/invite.rs

```diff
@@ -0,0 +1,75 @@
+/*! Invite message struct.
```

Comment:
> src/toxcore/group/packet/invite.rs

Maybe we should move it into src/toxcore/messenger/packet/group/invite.rs ?

Reply:
done